helpers/base.py

```diff
@@ -23,6 +23,11 @@ def wait_and_click(self, locator_type, locator):
         self.wait.until(EC.element_to_be_clickable((locator_type, locator)))
         self.driver.find_element(by=locator_type, value=locator).click()
 
+    def wait_and_send_keys(self, locator_type, locator, text):
```

Comment:
what do you think about merge `fill_input_field` and `wait_and_send_keys` as they do same thing ?
I understand that they use different waiters, but maybe stronger one is a good choice for every input field ?)